UpdateStatus.sh

```diff
@@ -1,30 +1,76 @@
 #!/bin/bash
 
+################################
+##---- CHANGE ME
+user_email="jalmond@cern.ch"
+#website_dir="/eos/user/" 
+website_dir="/afs/cern.ch/user/" 
+################################
+
 firstletter=`echo $USER | head -c 1`
 mkdir -p $SKFlatTag
-mkdir -p /eos/user/$firstletter/$USER/www/SKFlat/ProductionStatus/$SKFlatTag/
+mkdir -p $website_dir$firstletter/$USER/www/SKFlat/ProductionStatus/$SKFlatTag/
+################################                                                                                                                                                                            
+
+##-- setup crab3 since crab status and resubmittion is needed
+source /cvmfs/cms.cern.ch/crab3/crab.sh
+
+if [ -f "DoneSamples_"+$SKFlatTag+".txt" ];
```

Comment:
I assume this is for the case when some jobs failed and need to be resubmitted? I was thinking of having this file as a JSON format, and add timestatmp into it. I will update this later and remove this line.